integration-tests/src/lib.rs

```diff
@@ -460,7 +458,7 @@ fn reserve_transfer_native_asset_from_para_to_system_para() {
 	let receiver_balance_after = test.receiver.balance;
 
 	let delivery_fees = PopNetworkPara::execute_with(|| {
-		xcm_helpers::transfer_assets_delivery_fees::<
+		xcm_helpers::teleport_assets_delivery_fees::<
```

Comment:
I don't get why we are teleporting now and before we were doing a reserve transfer?

Pop Network would never be allowed to teleport DOT no?

Reply:
`transfer_assets_delivery_fees` is removed since `asset-test-utils v11.0.0`. 

Reference: https://docs.rs/asset-test-utils/11.0.0/asset_test_utils/xcm_helpers/index.html